### PR TITLE
Templates for pull requests and issues

### PR DESCRIPTION
**Pull request by alfredclwong**
worth a try :)

The issue templates are GitHub defaults.

The pull request template is from https://github.com/embeddedartistry/templates, which  has a lot of other good stuff.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (friggog) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Might be worth removing this

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (friggog) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: And changing this to incl Julia version etc

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (alfredclwong) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: maybe one day eh 😅

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (alfredclwong) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: It's now got OS, Julia version and OpticSim.jl version. Any further suggestions? @BrianGun @galran

**Review by BrianGun — APPROVED** [review]

**Pull request merged**